lib/agent/providers/geo/strategies.js

```diff
@@ -46,42 +46,20 @@ function wifi(cb) {
 
```

Comment:
let's take the opportunity to document this file

Reply:
Already done! Thank you!